Data/Repositories/CandidateRepository.cs

```diff
@@ -1,14 +1,25 @@
-﻿
+﻿using Data.Entities;
+
 namespace Data.Repositories
 {
-    public class CandidateRepository 
+    public class CandidateRepository : BaseRepository<Candidate>
     {
-        public void GetCandidatetById(int id) { }
+        public CandidateRepository(DataContext context) : base(context)
+        {
+
+        }
 
-        public void GetAllCandidate() { }
+        public Candidate GetCandidateById(int id) 
```

Comment:
You already have this logic in the base repository. There's no need to create method to use it